
The usual gcc-3.4 fixes. To those removing of -Wmissing-prototypes belongs,
but since those are spread over so many files I use th gcc wrapper for this.

  - Rene Rebe <rene@rocklinux.org>

--- ./kdirstat/kdirstatapp.h.orig	2003-08-26 22:47:47.000000000 +0200
+++ ./kdirstat/kdirstatapp.h	2004-05-07 19:52:50.088330112 +0200
@@ -46,7 +46,7 @@
     class KSettingsDialog;
     class KTreemapView;
     class KTreemapTile;
-};
+}
 
 using namespace KDirStat;
 
